test/loopback-dl.test.js

```diff
@@ -1473,6 +1483,20 @@ describe('DataAccessObject', function() {
     assert(error, 'An error should have been thrown');
   });
 
+  it('should throw when malformed logical operators follow valid logical clauses', function() {
```

Comment:
@Amir-61 Those guidelines were carefully documented by @bajtos as how we would like to move forward. That said, the files in our current codebase does not agree -- ie. we didn't submit PRs to all 60+ LB projects to fix this when creating this guideline. The problem is users tend to follow the styles already existing in the current codebase as best practice in general when submitting PRs to any project. So IMO, the only way to "ease" in the rule is to relax the requirement of forcing everyone to clean our house before we even do it -- so we don't end up with two styles in one file too (of course some of this has happened already -- we should fix those ones too).